astroid/node_classes.py

```diff
@@ -791,7 +788,7 @@ def repr_tree(
         indent="   ",
         max_depth=0,
         max_width=80,
-    ):
+    ) -> str:
```

Comment:
I wouldn't start with type annotations just yet. Chances are you're not going to finish for a while :D